aio/deploy/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -184,6 +184,14 @@ settings: {}
   ## Hide all access denied warnings in the notification panel
   # disableAccessDeniedNotifications: false
 
+## Pinned CRDs in dashboard
+pinnedCRDs: []
```

Comment:
Have you tested this? I think `namespaced` information is missing.

```json
{\"kind\":\"customresourcedefinition\",\"name\":\"clusters.kubermatic.k8s.io\",\"displayName\":\"Cluster\",\"namespaced\":false}
```

Reply:
It works, just does not add that "N" next to resource. Anyway added to the example.

Reply:
Ye, this is quite useful to have the visual indicator of whether or not a resource is namespaced.

Reply:
Sure, that was just a sidenote :slightly_smiling_face: